stripe/_api_resource.py

```diff
@@ -90,12 +117,10 @@ def _request(
         else:
             return obj
 
-    # The `method_` and `url_` arguments are suffixed with an underscore to
-    # avoid conflicting with actual request parameters in `params`.
     def _request_and_refresh(
         self,
-        method_: Literal["get", "post", "delete"],
-        url_: str,
+        method: Literal["get", "post", "delete"],
+        url: str,
         params: Optional[Mapping[str, Any]] = None,
         _usage: Optional[List[str]] = None,
```

Comment:
nit: if you are renaming parameters might as well rename the `_usage`, it's on a private method now.

Reply:
Done in [rename _usage -> usage](https://github.com/stripe/stripe-python/pull/1288/commits/1436ce40384b1a9ce7e5a936cf2133131a31004b)